internal/pkg/mapper/sharedcode/codes/local_save.go

```diff
@@ -47,14 +48,9 @@ func (m *mapper) onRowLocalSave(row *model.ConfigRow, recipe *model.LocalSaveRec
 	// Remove "isDisabled" unnecessary value from "meta.json".
 	// Shared code is represented as config row
 	// and always contains `"isDisabled": false` in metadata.
-	metaFile, err := recipe.Files.ObjectMetaFile()
-	if err != nil {
-		panic(err)
-	}
-	if value, found := metaFile.Content.Get(`isDisabled`); found {
-		if v, ok := value.(bool); ok && !v {
-			// Found `"isDisabled": false` -> delete
-			metaFile.Content.Delete(`isDisabled`)
-		}
+	if !row.IsDisabled {
+		// isDisabled == false -> hide "isDisabled" field in meta.json
+		fields, _ := recipe.Annotations[corefiles.HideMetaFileFieldsAnnotation].([]string)
+		recipe.Annotations[corefiles.HideMetaFileFieldsAnnotation] = append(fields, `isDisabled`)
```

Comment:
Shared code mapper skryva z `meta.json` field `isDisabled`, lebo pre sharedcodes ma vzdy hodnotu `false`.
Kedze su nahackovane pomocou config rows.

Predtym `save` mohol menit `meta.json` priamo, to uz nie je mozne.
Potrebujeme nejak povedat `corefiles` mapper-u aby field skryl.
Na to pouzijeme nove `Annotations`.
Takto kazdy mapper robi svoju robotu, no je tu moznost modifikovat spravanie.